### ISSUE TITLE
Screen off shortcut don't working

### ISSUE THREAD

**Issue by Poison5820** [seerge/g-helper#2207]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

After updating to version 0.158, turning off the screen via fn+f6 on the asus tuf fx505du stopped working. Likewise, it doesn’t work at 0.159. It worked up to version 0.154 inclusive.

### How to reproduce the bug?

always on 0.158

### Logs

[log.txt](https://github.com/seerge/g-helper/files/14573768/log.txt)

### Device and Model

asus tuf fx 505du

### Armoury Crate

Uninstalled

### Asus Services

0

### Version

0.158 and newer

### OS

windows 11 23h2

**Comment by seerge** [comment]
@Poison5820 hello. 

I have replaced old Windows method with a native ASUS one cause someone complained, that windows approach was causing display to flicker few times before turning off.

Apparently old devices don't support Asus method, so this build should still use old way for your device
[GHelper.zip](https://github.com/seerge/g-helper/files/14578271/GHelper.zip)

**Comment by seerge** [comment]
[GHelper.zip](https://github.com/seerge/g-helper/files/14579120/GHelper.zip)

**Comment by Poison5820** [comment]
Indeed, in the old method the screen flickered, but it’s better this way

**Comment by seerge** [comment]
@Poison5820 I actually have something else to test for you,

Can  you check if in this build screen toggle hotkey works?
[GHelper.zip](https://github.com/seerge/g-helper/files/14594715/GHelper.zip)

**Comment by Poison5820** [comment]
I checked this one, but it also doesn't work. Although it works in the pre-release version 0.160, but also with flickering

**Comment by seerge** [comment]
@Poison5820 oke, thanks for checking.

0.160 includes my "fix" from above :) and in last build it tried of another "asus" command works, apparently it doesn't.

How does screen-toggle hotkey work if you would start asus services (and quit g-helper), does it flicker ?

**Comment by Poison5820** [comment]
no, there is no flicker when the services are enabled

**Comment by seerge** [comment]
@Poison5820 then you can follow the steps from the guide. With the exception, you don't even need AC, you just need to start asus services and record what screen-off hotkey does in IRPMon :)

**Comment by Poison5820** [comment]
[screen.log](https://github.com/seerge/g-helper/files/14599682/screen.log)

**Comment by Poison5820** [comment]
I hope this is it

**Comment by seerge** [comment]
@Poison5820 sounds promising

So it's a command ``DEVS 0x050021`` and then it sends either ``0x03`` or ``0x83`` as a param (i assume you toggled it ON / OFF twice)

Good sign is that HOTKEY (Fn+F6) produces keycode ``0x83`` as well, so looks like match ...

can you check this build ?
[GHelper.zip](https://github.com/seerge/g-helper/files/14599818/GHelper.zip)

**Comment by Poison5820** [comment]
not working

**Comment by Poison5820** [comment]
It doesn't turn off at all

**Comment by seerge** [comment]
@Poison5820 can you post a log from g-helper?

**Comment by Poison5820** [comment]
what is the path of the log

**Comment by Poison5820** [comment]
at the same time, the backlight of the keyboard is turned off

**Comment by Poison5820** [comment]
but the screen doesn't turn off

**Comment by seerge** [comment]
@Poison5820 ok, it's a disappointment :) Apparently you have intercepted backlight control then. As that's the only command available in the log.

I have restored previous method for switching display off again, unfortunately there is no other known ways to do it :)

[GHelper.zip](https://github.com/seerge/g-helper/files/14599937/GHelper.zip)

**Comment by Poison5820** [comment]
If that's how it works, then let it
